wiki/en/Tips-Tricks-More.md

```diff
@@ -80,21 +80,92 @@ Here is the script:
+     |--------:|---------|
+     | `f` | Fader |
+     | `p` | Pan   |
+     | `s` | Solo  |
+     | `m` | Mute  |
+
+   * `offset` is the base MIDI CC number for the control.
+
+   * `count` is the number of CC values for the control, defaulting to 1 (i.e. the number Jamulus channels that can be controlled).
+
+   * Two additional `control letter` values are available:
+
+     1. `o` controls Mute Myself and has a single `offset` (i.e. `count` is ignored and taken as 1).
+
+     2. `z` is an option to reserve the first CC number for a control to mean "my Jamulus channel".  For example
+
+        ```
+        --ctrlmidich "1;f0*9;z"
+        ```
+
+        would mean MIDI CC0 controls "my" Jamulus channel fader, with CC1 to CC8 for Jamulus channels 0 to 7 (so if you were channel 3, you get two MIDI CC controls - CC0 and CC4).  (Remember that this will *not* affect the sound others hear.)
+
+   An example for a Korg nanoKONTROL2 using eight sliders starting at MIDI CC 0 to control faders and eight knobs starting at MIDI CC 16 to control pan, on any MIDI channnel, for eight Jamulus channels would be
+
+   ```
+   --ctrlmidich "0;f0*8;p16*8"
+   ```
+
+   Another example:
+
+   ```
+   --ctrlmidich "1;f0*9;p16*9;s32*9;m48*9;o64;z"
+   ```
+
+   Here, Jamulus listens on MIDI channel 1 and uses nine MIDI controller numbers for each Jamulus control: one for "My Channel" plus eight for Jamulus channels 0 to 7.  MIDI CC numbers 0 (for "My Channel") to 8 control faders.  Pan controls start at CC number 16 ("my") and end at 24; Solo 32 ("my") to 40 and Mute 48 ("my") to 56; Mute Myself is enabled/disabled by CC number 64.
+
+Please note that buttons on your MIDI controller need to be set to "toggle" mode. This means that when pressed to 'turn on' a control, it must send a MIDI CC number with a value >=64, and to 'turn off' the control it must send the same CC number with a value <64. You can read your controller's manual to find out how to set this.
+
+*Note*: Jamulus does not provide feedback on the on/off state of buttons, meaning that your controller must keep track and toggle LEDs (if any) to 'on' or 'off' itself.
+
+Jamulus channel numbers are assigned by the server and displayed in the mixer when you use `--ctrlmidich`.  When you join a server, you can get a different Jamulus channel depending on who else is already there (hence using the "my channel" option, `z`).
 
 *Tip*: When you enable MIDI control in Jamulus, each user's name is prepended with a number, with the leftmost user starting at 0, then 1, etc. With default settings, when some users leave and others join, their left-right arrangement in the GUI may cease to follow a numerical order, making it more difficult to know who each physical fader/knob on your MIDI controller corresponds to. In order to keep the fader strips following a numerical order, go to "View" on the top menu bar and switch between "No User Sorting" and another option and then back again (e.g. type `Ctrl+N`, `Ctrl+O`).
```

Comment:
I think the style guide implies we should use the word "fader" here and not "user", because it says a user could have more than one fader/channel. Although saying "when some users leave and others join" is OK, I assume.

Reply:
Could you state more clearly which instance of the word "fader" you're referring to.

I'll just point out that we need to distinguish between the UI widget that adjusts the volume level and the hardware component that emits a MIDI message and the Jamulus channel number assigned by the server to a participant that has volume, pan and mute widgets in the UI.

Reply:
(Also, I don't know if the para is strictly correct.  It would probably be useful to have a "sort by channel" option available when `--ctrlmidich` was in use, too.)

Reply:
On second thoughts, I'm not sure. The style guide clarifies the differences between "channel", "fader" and "person", so I was wondering if here it should be, "When you enable MIDI control in Jamulus, each user's fader is prepended with a number,..."

Reply:
That's still nonsense.  The fader isn't.  It's in front of the name at the bottom of the mixer channel.